recursio-client/Characters/Player.tscn

```diff
@@ -59,11 +51,11 @@ script = ExtResource( 2 )
 
 [node name="Selector" type="Spatial" parent="KinematicBody" index="0"]
 transform = Transform( -4.37114e-08, 0, 1, 0, 1, 0, -1, 0, -4.37114e-08, 0, 1.27928, 0 )
+visible = false
```

Comment:
could we remove the selector completely rather than just hiding it?

Reply:
yes :D